typing_extensions/src_py3/typing_extensions.py

```diff
@@ -1653,6 +1661,11 @@ def __eq__(self, other):
         def __hash__(self):
             return hash((self.__origin__, self.__metadata__))
 
+        def __instancecheck__(self, obj):
+            raise TypeError("Annotated cannot be used with isinstance().")
+
+        def __subclasscheck__(self, cls):
+            raise TypeError("Annotated cannot be used with issubclass().")
```

Comment:
Why are these needed? To tweak the error message? I think this would be an error already with the inherited ones. Please add a comment.

Reply:
You marked this as resolved, but nothing has changed here, only in the other version (where it is actually more clear).

Reply:
Oh, sorry. I thought I had added a comment: without this we get an error about "subscripted generics" which I thought was a bit confusing. Come to think of it though I've probably added too much code to control what error messages get thrown. Both this and get_cons can be simplified.